### PR TITLE
Cleanup Rubocop warnings for spec files

### PR DESCRIPTION
**Pull request by pawelchcki**
I've found out spec files are not following Rubocop linting configuration. 

This PR aligns the code in spec with Rubocop configuration. Its mostly a product of running `rubocop -a ` in a loop.

### CODE REVIEW

File: spec/ddtrace/quantization/http_spec.rb [hunk 1]
```diff
@@ -47,7 +48,7 @@
 
       context 'with Unicode characters' do
         # URLs do not permit unencoded non-ASCII characters in the URL.
-        let(:url) { "http://example.com/path?繋がってて" }
+        let(:url) { 'http://example.com/path?繋がってて' }
```
[the remark this answers is not in the document]
Comment (pawelchcki) [reply spec/ddtrace/quantization/http_spec.rb]: I think only if you are escaping them via `/u` this change seems to also pass all tests. So I'd assume its safe.

**Comment by pawelchcki** [comment]
@delner Is it ok to merge this into `0.13-dev`?

**Comment by palazzem** [comment]
@pawelchcki I think you can merge that on `master` since it impacts only the test suite. Then you can merge the current master in `0.13-dev` to keep it in sync. This avoids conflicts if contributors start from `dev` or `master`.

**Comment by pawelchcki** [comment]
@palazzem Thanks!
I've remade the changes for master, and I'll soon post PR merging both into 0.13-dev

**Comment by pawelchcki** [comment]
@delner Can you review this PR? I've since rebased this PR on master, and changes only correspond to those required by master. I've created separate PR for 0.13-dev changes

**Pull request merged**